server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfig.java

```diff
@@ -875,6 +875,18 @@ public class IoTDBConfig {
   /** Thread keep alive time in ms of data block manager. */
   private int dataBlockManagerKeepAliveTimeInMs = 1000;
 
+  /** Thrift socket and connection timeout between data node and config node. */
+  private int connectionTimeoutInMS = (int) TimeUnit.SECONDS.toMillis(20);
+
+  /**
+   * ClientPool will have so many selector threads (TAsyncClientManager) to distribute to its
+   * clients.
+   */
+  private final int selectorNumOfClientPool =
+      Runtime.getRuntime().availableProcessors() / 4 > 0
+          ? Runtime.getRuntime().availableProcessors() / 4
+          : 1;
+
```

Comment:
I feel it would be better to use "connectionTimeoutInMS" as a global configuration in iotdb-confignode.properties and "selectorNumOfClientPool" as a parameter that can be configured independently for each standalone machine.

Reply:
I personally feel that no one will adjust it in general. If we want to place it in ConfigNode, we should open another issue to sort out the configuration list that Datanode should be placed in ConfigNode. I feel that this PR is not necessary to do this